docs/sources/migration-guide/migrating-from-thanos-or-prometheus.md

```diff
@@ -68,7 +68,9 @@ When the Thanos sidecar uploads blocks, it includes the external labels from Pro
 When you query the block, Thanos injects Prometheus’ external labels in the series returned in the query result.
 Thanos also uses labels for the deduplication of replicated data.
 
-If you want to use existing blocks from Thanos by Grafana Mimir, there are some considerations:
+An alternative to migrating the blocks is to continue to run Thanos and Mimir in parallel. You can direct clients (ie. Grafana) to Thanos and then setup Mimir as a storage node within Thanos. This allows you to query historical data from the Thanos storegateway along with current data being pushed into Mimir from your Prometheus servers. You will need to carefully consider the external labels Mimir will present to Thanos along with the external labels that currently exist in Thanos to ensure compatibility with existing dashboards. Once the Thanos retention period has passed you can then tranistion all clients to query Grafana Mimir directly. This approach is very much experimental and has not yet seen significant use in production. For a write-up please see this link [here](https://gist.github.com/gmintoco/e949cb9bf12e427449b36d6897239bcf).
```

Comment:
I'm not sure this is right place in the doc to add your method, cause this section is "Considerations on Thanos specific features" and I'm not sure an alternative migration path fits well into it.

Would you be open to actually create a new doc page (e.g. "Migrating from Thanos to Mimir using Thanos sidecar") and actually importing all your gist content into the new page? In the new doc page you could state it's experimental and not battle tested.

Reply:
yep that makes sense